lib/net/x86-64-linux/Net.v3

```diff
@@ -0,0 +1,371 @@
+// Copyright 2022 Kunal Sareen. All rights reserved.
```

Comment:
Can you make this copyright the Virgil authors?

Reply:
Done.